### ISSUE TITLE
Lualine and notifications is not working

### ISSUE THREAD

**Issue by augustocdias** [wyattjsmith1/weather.nvim#6]
I set up like this

```lua
-- weather config
require('weather').setup({
    openweathermap = {
        app_id = {
            var_name = 'WEATHER_TOKEN',
        },
    },
    weather_icons = require('weather.other_icons').nerd_font,
})
require('weather.notify').start(70, 'info')
```

I have nvim-notify installed and I'm not getting any notification nor the info is being shown on lualine.

My lualine is configured like this:

```lua
local lualine = require('lualine')

lualine.setup({
    options = {
        icons_enabled = true,
        theme = 'tokyonight',
        section_separators = { left = '', right = '' },
        component_separators = { left = '', right = '' },
        disabled_filetypes = {},
    },
    sections = {
        lualine_a = { 'mode' },
        lualine_b = { 'diff', 'branch' },
        lualine_c = {
            {
                'filename',
                file_status = true, -- displays file status (readonly status, modified status)
                path = 1, -- 0 = just filename, 1 = relative path, 2 = absolute path
            },
            {
                function()
                    return require('lsp-status').status()
                end,
            },
        },
        lualine_x = {
            'encoding',
            'filetype',
        },
        lualine_y = { require('weather.lualine').default_c, 'progress' },
        lualine_z = {
            'location',
            {
                'diagnostics',
                sources = { 'nvim_diagnostic' },
                symbols = { error = ' ', warn = ' ', info = ' ', hint = ' ' },
            },
        },
    },
    inactive_sections = {
        lualine_a = {},
        lualine_b = {},
        lualine_c = {
            {
                'filename',
                file_status = true, -- displays file status (readonly status, modified status)
                path = 1, -- 0 = just filename, 1 = relative path, 2 = absolute path
            },
        },
        lualine_x = { 'location' },
        lualine_y = {},
        lualine_z = {},
    },
    tabline = {},
    extensions = { 'quickfix', 'nvim-tree', 'fzf' },
})
```

**Comment by wyattjsmith1** [comment]
Closed in #3 .

**Comment by wyattjsmith1** [comment]
Hi, sorry about that. I closed this by mistake. 

Thank you for your report, I see 2 issues:
1. **Lualine isn't working** - Unfortunately, I modified the api after writing the README, and forgot to update it. `default_c/f` is a function which should be invoked like `require('weather.lualine').default_c()`. That function accepts a table with the following keys:
- `pending`: Text to display while fetching results
- `error`: Text to display when there is an error.
Both fields are optional (you don't need to pass anything).
I will update the readme immediately. Thanks for this report.
2. **Notifications aren't working:** Notifications only display notifications from official sources (e.g. national weather service). There aren't notifications for regular updates. Are you expecting any of these alerts??
